### PR TITLE
Update tr.po

### PR DESCRIPTION
**Pull request by bitigchi**
Updated Turkish translations.

### CODE REVIEW

**Comment by Minoru** [comment]
Hi, thanks for working on this! Most of CI builds are broken because of [some syntax error on line 602](https://travis-ci.com/github/newsboat/newsboat/jobs/379137352#L836) — can you please fix that?

File: po/tr.po [hunk 1]
```diff
@@ -577,19 +585,22 @@ msgstr "Hiçbir besleme seçilmedi!"
...
 msgid "ftauln"
-msgstr ""
+msgstr "ibyosh"
 
 #: src/feedlistformaction.cpp:156
 msgid ""
 "Sort by (f)irsttag/(t)itle/(a)rticlecount/(u)nreadarticlecount/(l)astupdated/"
 "(n)one?"
-msgstr ""
+msgstr "Sırala: (i)lketiket/(b)aşlıksayısı/(y)azısayısı/(o)kunmamışyazısayısı"
```
Comment (Minoru) [inline po/tr.po:594]: This line is missing a slash at the end.

File: po/tr.po [hunk 2]
```diff
@@ -577,19 +585,22 @@ msgstr "Hiçbir besleme seçilmedi!"
...
+"(s)ongüncelleme/(h)içbiri"
 
 #: src/feedlistformaction.cpp:196
 msgid ""
 "Reverse Sort by (f)irsttag/(t)itle/(a)rticlecount/(u)nreadarticlecount/"
 "(l)astupdated/(n)one?"
 msgstr ""
+msgstr "Ters Sırala: (i)lketiket/(b)aşlıksayısı/(y)azısayısı/(o)kunmamışyazısayısı"
```
Comment (Minoru) [inline po/tr.po:602]: This line is missing a slash at the end as well.

File: po/tr.po [hunk 3]
```diff
@@ -940,22 +950,24 @@ msgstr "Yukarıdakilerin tümü okundu olarak imleniyor..."
...
 #: src/itemlistformaction.cpp:677 src/itemlistformaction.cpp:715
 msgid "dtfalgr"
-msgstr "dtfalgr"
+msgstr "tbaylgr"
 
 #: src/itemlistformaction.cpp:679
 msgid "Sort by (d)ate/(t)itle/(f)lags/(a)uthor/(l)ink/(g)uid/(r)andom?"
-msgstr ""
+msgstr "Sırala: (t)arih/(b)aşlık/b(a)yraklar/(y)azar/bağ(l)antı/(g)uid/"
+"(r)astgele"
```
Comment (Minoru) [inline po/tr.po:965]: Just checking: shouldn't there be a question mark at the end here?

File: po/tr.po [hunk 4]
```diff
@@ -940,22 +950,24 @@ msgstr "Yukarıdakilerin tümü okundu olarak imleniyor..."
...
 
 #: src/itemlistformaction.cpp:717
 msgid "Reverse Sort by (d)ate/(t)itle/(f)lags/(a)uthor/(l)ink/(g)uid/(r)andom?"
-msgstr ""
+msgstr "Ters Sırala: (t)arih/(b)aşlık/b(a)yraklar/(y)azar/bağ(l)antı/(g)uid/"
+"(r)astgele"
```
Comment (Minoru) [inline po/tr.po:970]: Just checking: should there be a question mark at the end here?

File: po/tr.po [hunk 5]
```diff
@@ -1738,23 +1754,23 @@ msgstr ""
...
 msgid "regcomp returned code %i"
-msgstr ""
+msgstr "regcomp %i kodunu döndürdü"
 
 #: rust/regex-rs/src/lib.rs:243 rust/regex-rs/src/lib.rs:247
 msgid "regexec returned code %i"
-msgstr ""
+msgstr "regexec %i kodunu döndürdü"
 
 #~ msgid "config"
```
Comment (Minoru) [inline po/tr.po:1775]: Please remove those, these are just leftovers of previous edits.

Comment (bitigchi) [reply po/tr.po:1775]: I am not sure what to remove, the whole string?

**Comment by bitigchi** [comment]
> 
> 
> Hi, thanks for working on this! Most of CI builds are broken because of [some syntax error on line 602](https://travis-ci.com/github/newsboat/newsboat/jobs/379137352#L836) — can you please fix that?

Fixed.

File: po/tr.po (hunk 5 above)
Comment (Minoru) [reply po/tr.po:1775]: I mean the lines that start with `#~`.

**Comment by Minoru** [comment]
JFYI the FreeBSD 11.4 and Travis coverage builds are failing for an unrelated reason. I'll submit a PR fixing those in a couple minutes.

**Review by Minoru — APPROVED** [review]
Looks good :+1: I'll merge once CI builds finish. Thanks for updating the translation!

**Pull request merged**